libc/unistd/setgid.c

```diff
@@ -0,0 +1,9 @@
+#include <syscall.h>
+#include <syscall_nums.h>
+#include <sys/types.h>
+
+DEFN_SYSCALL1(setgid, SYS_SETGID, unsigned int);
+
+int setgid(gid_t gid) {
+	return syscall_setgid(gid);
```

Comment:
I know it's missing from the `uid` counterparts, but these should handle `errno`.